tools/release_do_nothing.py

```diff
@@ -327,7 +327,7 @@ def finalise_whats_new(
 
     message = (
         "Commit and push all the What's New changes.\n"
-        f'git commit -am "Whats new updates for {release_strings.tag} .";\n'
```

Comment:
I guess this is your own addition, since AFAICT codespell only checks docstrings + comments, not code, right ??
( asking for a friend )

TBH their docs don't make this crystal clear.